SrcDragonfly/masterParallel.m

```diff
@@ -1,4 +1,4 @@
-function [fOutVar,nBlockPerCPU, totCPU] = masterParallel(Parallel,fBlock,nBlock,NamFileInput,fname,fInputVar,fGlobalVar,Parallel_info,initialize)
+function [fOutVar,nBlockPerCPU,CPUforIndex,totCPU] = masterParallel(Parallel,fBlock,nBlock,NamFileInput,fname,fInputVar,fGlobalVar,Parallel_info,initialize)
```

Comment:
make a separate commit. 
There are many lines associated to HPC, but others that fix calls, mostly to octave.